### PR TITLE
Refining Repository Management's UI.

### PR DESCRIPTION
**Pull request by dhoehna**
## Summary of the pull request
My original plan was to use a feature branch and push all my changes to that.  But I already messed it up (on my computer.  I spent a good while un-doing my mistake).

Going forward Repository Management is in experimental, visible only on dev builds, and incremental changes will go to main.

## Detailed description of the pull request / Additional comments

In this PR I did modify a few files
1. App.xaml.cs - to use the Repository Management extension.
2. RepositoryManagementMainPageView - fixing margins to better match FIGMA.
3. NavConfig.jsonc - Add Repository Management to experimental.  The tool is off by default.

Additionally, I reverted the commit that changed LandingPage.xaml.  The version in this PR matches the version in Main. 

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/SetupFlow/DevHome.SetupFlow/Extensions/ServiceExtensions.cs
Comment (krschau) [inline tools/SetupFlow/DevHome.SetupFlow/Extensions/ServiceExtensions.cs:7]: Leftover?

File: tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj
Comment (krschau) [inline tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj:2]: csproj files should have indents of 2 spaces. This used to happen by default but now a couple PRs have come in in a few days where they've had 4. I'll look into what changed but let's keep 2 since that's what all the other files have.

File: src/App.xaml
Comment (krschau) [inline src/App.xaml:18]: nit: I'm not sure if we're going to go with the 2 space buffers as standard, so I'd rather not change this now.

File: src/DevHome.csproj
Comment (krschau) [inline src/DevHome.csproj:106]: Would be great to add this to the architecture diagram in architecture.md

**Comment by dhoehna** [comment]
After significant push back This PR will be closed and the branch user/dhoehna/RepoManagement will be the feature branch.

File: tools/SetupFlow/DevHome.SetupFlow/Extensions/ServiceExtensions.cs
Comment (dhoehna) [reply tools/SetupFlow/DevHome.SetupFlow/Extensions/ServiceExtensions.cs:7]: Removed.

File: tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj
Comment (dhoehna) [reply tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj:2]: My theory is that I did modify this in notepad++ and I have indents of 4 spaces.  I'll get this changed to 2.

File: src/App.xaml
Comment (dhoehna) [reply src/App.xaml:18]: This is the work of the xaml formatter extension.  It adds the second space on a save.  I'll get this fixed.

File: src/DevHome.csproj
Comment (dhoehna) [reply src/DevHome.csproj:106]: Done.

File: settings/DevHome.Settings/Strings/en-us/Resources.resw
Comment (dhoehna) [reply settings/DevHome.Settings/Strings/en-us/Resources.resw:29]: I can take a look.

File: tools/RepositoryManagement/DevHome.RepositoryManagement/ViewModels/RepositoryManagementMainPageViewModel.cs
Comment (dhoehna) [reply tools/RepositoryManagement/DevHome.RepositoryManagement/ViewModels/RepositoryManagementMainPageViewModel.cs:28]: Done.

File: tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj
Comment (krschau) [inline tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj:34]: Empty, can be removed
```suggestion
```

File: tools/RepositoryManagement/DevHome.RepositoryManagement/Extensions/ServiceExtensions.cs
Comment (krschau) [inline tools/RepositoryManagement/DevHome.RepositoryManagement/Extensions/ServiceExtensions.cs:8]: Can be deleted
```suggestion
```

File: tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml
Comment (krschau) [inline tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml:50]: Does this need to be repeated here and down below?

File: docs/architecture.md
Comment (krschau) [inline docs/architecture.md:85]: Please also add the three new package references in the mermaid diagram above.

File: tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml
Comment (dhoehna) [reply tools/RepositoryManagement/DevHome.RepositoryManagement/Views/RepositoryManagementMainPageView.xaml]: It does not.  I believe this is a hold off from when I added it to the resources.  I removed the dupe from Grid.Resources.

File: docs/architecture.md
Comment (dhoehna) [reply docs/architecture.md:86]: I believe I got all the project references.  I might have had 3 before.  Now I have only 2.

File: tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj
Comment (DefaultRyan) [inline tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj:31]: Does this icon mean the file is missing a newline at the end?

File: tools/SetupFlow/DevHome.SetupFlow/DevHome.SetupFlow.csproj
Comment (krschau) [inline tools/SetupFlow/DevHome.SetupFlow/DevHome.SetupFlow.csproj:24]: Is this project reference used?

**Review by krschau — APPROVED** [review]
Approved pending one question

**Review by DefaultRyan — APPROVED** [review]

File: tools/SetupFlow/DevHome.SetupFlow/DevHome.SetupFlow.csproj
Comment (dhoehna) [reply tools/SetupFlow/DevHome.SetupFlow/DevHome.SetupFlow.csproj]: You are correct.  Removed.

**Pull request merged**
